### DISCUSSION TITLE
This was really nice!

### DISCUSSION THREAD

**Discussion by ghost** [nasirkhan/laravel-starter#329] [General]
thank you for building this amazing template

**Comment by ghost** [comment]
but some features of this template are not responsive on mobile 

![image](https://user-images.githubusercontent.com/45863605/168463411-77c78121-738a-4e99-9d59-1ef3848dcf7a.png)

**Comment by nasirkhan** [comment]
Thanks for mentioning this issue. While updating this project I started to
create this new theme using tailwind. I know all the parts are not
responsive. I will try to fix those soon. If you already fixed the issues,
iou submit a PR.


Nasir Khan Saikat
https://nasirkhn.com
